R/module_teal.R

```diff
@@ -182,28 +166,47 @@ srv_teal <- function(id, data, modules, filter = teal_slices()) {
     }
 
 
-
     if (inherits(data, "teal_data_module")) {
       setBookmarkExclude(c("teal_modules-active_tab"))
-      bslib::nav_insert(
-        id = "teal_modules-active_tab",
-        position = "before",
-        select = TRUE,
-        bslib::nav_panel(
-          title = icon("fas fa-database"),
-          value = "teal_data_module",
-          tags$div(
-            ui_init_data(session$ns("data")),
-            validate_ui
+      insertUI(
+        selector = ".teal-modules-wrapper .nav-item-custom",
+        where = "beforeBegin",
+        actionButton(session$ns("open_teal_data_module_ui"), NULL, icon = icon("database"))
```

Comment:
<img width="1512" alt="image" src="https://github.com/user-attachments/assets/28796bd0-689f-414a-8d4d-aa91fab107dd" />

- Data button is the only icon with the boarder around element. 
- "Modules" element is blue unlike the rest
- Breadcrumb to be .text-secondary and maybe smaller than "Modules"?